src/runtime/basic-runtime.js

```diff
@@ -228,6 +228,7 @@ export class BasicRuntime {
     this.setupAndShowAutoPrompt({
       autoPromptType,
       alwaysShow,
+      isAccessibleForFree,
```

Comment:
Could you update `src/api/basic-subscriptions.ts` too?

Reply:
Done.